libreoffice-starter-extension-master/registry/org/openoffice/Office/Accelerators.xcu

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<oor:component-data xmlns:oor="http://openoffice.org/2001/registry"
+	xmlns:xs="http://www.w3.org/2001/XMLSchema" oor:name="Accelerators"
+	oor:package="org.openoffice.Office">
+	<node oor:name="PrimaryKeys">
+		<node oor:name="Modules">
+			<node oor:name="com.sun.star.text.TextDocument">
+				<node oor:name="1_SHIFT_MOD1" oor:op="replace">
+					<prop oor:name="Command">
+						<value xml:lang="en-US">service:org.libreoffice.example.TildeTranslator?actionOne</value>
```

Comment:
`actionOne` un pārējos kā arī to klases vēlams nosaukt ar pašpaskaidrojošākiem nosaukumiem, piemēram, `actionOpenTranslator`, `actionReplaceWithTranslation`, `actionInsertTranslation`.